### ISSUE TITLE
Module migration: mod_menu end level is slightly too large

### ISSUE THREAD

**Issue by mahagr** [fastslack/jUpgrade#5]
Joomla 1.5 uses start/end level of 0 => 1 to show one level only.

Joomla 1.7 accepts the same start/end level, so the above values become 1 => 1 (not 1 => 2).

**Comment by fastslack** [comment]
@mahagr so all access should be 1?

**Comment by mahagr** [comment]
Sorry, I was talking about start and endlevel inside module parameters, which are used to determine which levels of the menu are shown:

Forum > Latest > User = 0,3 in Joomla 1.5 but 1,3 in Joomla 1.7.3.
Latest > user = 1,3 in Joomla 1.5 but 2,3 in Joomla 1.7.

My previous mistake was that both of the values increase, not just one.

**Comment by fastslack** [comment]
Fixed in commit 8ad806cf7255f2aae931621bb44586c887170bdf

``` php
diff --git a/trunk/admin/includes/migrate_modules.php b/trunk/admin/includes/migrate_modules.php
index 324ba5b..8e3af6a 100644
--- a/trunk/admin/includes/migrate_modules.php
+++ b/trunk/admin/includes/migrate_modules.php
@@ -61,7 +61,6 @@ class jUpgradeModules extends jUpgrade
        protected function convertParamsHook(&$object)
        {
                if (isset($object->startLevel)) $object->startLevel++;
-               if (!empty($object->endLevel)) $object->endLevel++;
        }

        /**
```

Thanks Matias
